First draft of NCMC post

maxentile opened the pull request:
Comments welcome! Aiming to publish by end of day Wednesday.

maxentile committed:
First draft of NCMC post

bas-rustenburg commented:
Two quick observations:

1. We'll have to use double dollar signs for math instead of single ones it seems. 

2. The blogging template does not seem to support multiple authors.

jchodera commented on _posts/2016-09-07-ncmc.md:
```diff
@@ -0,0 +1,125 @@
+For instance, sometimes information is available about a part of the system in isolation, or we know which parts of a potential induce the biggest barriers in our system.
+NCMC constructs valid acceptance criteria when we use non-equilibrium driven processes as proposals.
+This allows us to use the information we have to drive over or perturb barriers, and relax the system into equilibrium in a new state.
+As opposed to an instantaneous Monte Carlo switch, relaxation can drastically improve acceptance rates, increasing the efficiency of the algorithm.
```
It might be useful to point out that people have been using clever Monte Carlo schemes to speed up sampling for ages, but they suffer from the problem of low acceptance rates in systems where the parts of the system we know something useful about (e.g. a solute) are strongly coupled to the parts of the system we don't know about (e.g. solvent).

jchodera replied:
To me, the big advantage of NCMC is to take an instantaneous Monte Carlo move that works well for the component of the system in isolation and convert it into a nonequilibrium move that works well in the context of the component coupled to other things.

jchodera commented on _posts/2016-09-07-ncmc.md:
```diff
@@ -0,0 +1,125 @@
+
+![image](https://cloud.githubusercontent.com/assets/5759036/18291418/9dde2d54-7455-11e6-8be0-0fbae585767a.png)
+
+Our isolated "insight" into the system comes in the form of knowing the energy term for the bond.
```
We don't really need to know the energy term for the bond---just the fact that it has two minima with a big barrier in between, and where the minima are in relation to each other. That's enough to construct an efficient MC move for the dimer in isolation.

jchodera commented on _posts/2016-09-07-ncmc.md:
```diff
@@ -0,0 +1,125 @@
+In a vacuum, this insight serves us very well, reducing our correlation time from 60 to 0.
+
+![image](https://i.sli.mg/yOkHGO.jpg)
+
```
Might be worth noting how we are mixing in a bit of MC with Metropolized MD---that may not be obvious to people.

jchodera commented on _posts/2016-09-07-ncmc.md:
```diff
@@ -0,0 +1,125 @@
+
+![image](http://www.pnas.org/content/108/45/E1009/F4.large.jpg)
+
+As an analogy, imagine you're trying to push through a crowded subway station. If you instantly "teleported" to a random point in the crowd, you would probably have a bad time / end up overlapping with another commuter. If you try to sprint, you'll have to work a lot harder bowling people over. If you're patient and walk at roughly the same speed as everyone around you, you don't have to work as  ...
```
This is great!

jchodera commented:
I made some comments, but this is really great, guys!  Feel free to merge whenever you like.

maxentile committed:
Minor edits

Addressing some of Bas and John's comments:
- Switched $ -- > $$
- Reworded a few phrases
- Added a bit more context

maxentile commented:
Thanks, John!

bas-rustenburg committed:
Update 2016-09-07-ncmc.md

Author formatting

bas-rustenburg commented:
Post is live.
